TwoNote/app/src/main/java/com/microsoft/device/display/samples/twonote/ItemsListFragment.kt

```diff
@@ -16,31 +17,37 @@ import android.widget.ListView
 import androidx.fragment.app.Fragment
 import com.google.android.material.floatingactionbutton.FloatingActionButton
 import com.microsoft.device.display.samples.twonote.model.DataProvider
+import com.microsoft.device.display.samples.twonote.model.DirEntry
+import com.microsoft.device.display.samples.twonote.model.Inode
 import com.microsoft.device.display.samples.twonote.model.Note
 import com.microsoft.device.dualscreen.layout.ScreenHelper
+import java.io.*
 import java.time.LocalDateTime
 import kotlin.collections.ArrayList
 
 class ItemsListFragment : Fragment(), AdapterView.OnItemClickListener {
-    private var arrayAdapter: ArrayAdapter<Note>? = null
+    private var arrayAdapter: ArrayAdapter<Inode>? = null
```

Comment:
ok this is 1000% unnecessary but how would you feel about renaming the class "INode"

Reply:
1000% should do that